packages/xforms-engine/src/instance/text/markdownFormat.ts

```diff
@@ -208,7 +208,8 @@ function toOdkMarkdown(str: string): MarkdownNode[] {
 }
 
 export function format(chunks: readonly TextChunk[]): MarkdownNode[] {
-	outputStrings = new Map<string, string>();
 	const str = escapeEditableChunks(chunks);
-	return toOdkMarkdown(str);
+	const result = toOdkMarkdown(str);
+	outputStrings.clear();
```

Comment:
By clearing after the fact the output strings aren't kept in memory longer than necessary.

Also it saves on creating a new map.